app/rnx2rtkp/rnx2rtkp.c

```diff
@@ -155,7 +155,10 @@ int main(int argc, char **argv)
         else if (!strcmp(argv[i],"-b")) prcopt.soltype=1;
         else if (!strcmp(argv[i],"-c")) prcopt.soltype=2;
         else if (!strcmp(argv[i],"-i")) prcopt.modear=2;
-        else if (!strcmp(argv[i],"-h")) prcopt.modear=3;
+        else if (!strcmp(argv[i],"-h")) {
+            prcopt.modear=3;
```

Comment:
@tao-swift: this doesn't account for the possibility that the user may specify `-w` before `-h`.  With your approach, `modear` will first be set to 6 (==ARMODE_WL) due to the `-w` flag and will then change to 3 (==ARMODE_FIXHOLD) due to the `-h` flag.  This means that the widelane support will not be used.

Reply:
What is wrong with the approach which I proposed?  It correctly handles both the `-w -h` and `-h -w` cases (as well as `-h` separately from `-w`)

Reply:
Alternatively you could change the above to something like `if(prcopt.modear!=ARMODE_WL) prcopt.modear=ARMODE_FIXHOLD`

Reply:
yeah, you are right. Feel free to merge yours in. The alternative change that you mentioned ^ also looks good to me.

Probably not a good idea to use this fix and hold option, but we can include it in.